src/main/kotlin/blackjack/model/CardDeck.kt

```diff
@@ -0,0 +1,20 @@
+package blackjack.model
+
+class CardDeck {
+    var cards: List<Card> = INIT_CARDS
+        private set
+
+    fun drawCard(): Card {
+        val pickedCard = cards.shuffled().toMutableList()[0];
```

Comment:
```suggestion
        val pickedCard = cards.shuffled().first()
```
좀 더 간단하게 나타낼 수 있겠네요.
매번 shuffle을 할 지도 고민해보시면 좋을 것 같아요.